doc/API.md

```diff
@@ -102,6 +102,7 @@ JitsiMeetJS.setLogLevel(JitsiMeetJS.logLevels.ERROR);
         - AVAILABLE_DEVICES_CHANGED - notifies that available participant devices changed (camera or microphone was added or removed) (parameters - id(string), devices(JS object with 2 properties - audio(boolean), video(boolean)))
         - CONNECTION_STATS - New local connection statistics are received. (parameters - stats(object))
         - AUTH_STATUS_CHANGED - notifies that authentication is enabled or disabled, or local user authenticated (logged in). (parameters - isAuthEnabled(boolean), authIdentity(string))
+        - CONNECTION_CLOSED - Indicates that the connection to the conference has been closed. (Should be received after JitsiConference.leave())
```

Comment:
CONNECTION_DISCONNECTED is fired when the status of JitsiConnection is changed to disconnected (normally that happens when you call JitsiConnection.disconnect). CONNECTION_CLOSED was related to JitsiConference object. But I removed it because when I thought again about it, I realized that it shouldn't be useful outside the library since it indicates that only specific part of JitsiConference.leave process was finished successfully.